### PR TITLE
fixed cross domain ajax

### PR DESCRIPTION
**Pull request by tito10047**
fixed bug [bug:#178]

### CODE REVIEW

File: src/Tracy/assets/Bar/bar.js [hunk 1]
```diff
@@ -357,9 +357,22 @@
 		if (!layer.dataset.id) {
 			return;
 		}
+		var reURLInformation = new RegExp('^(https?:)//(([^:/?#]*)(?::([0-9]+))?)(/[^?#]*)');
+	        var oldOpen = XMLHttpRequest.prototype.open;
+	        XMLHttpRequest.prototype.open = function(){
```
Comment (dg) [inline src/Tracy/assets/Bar/bar.js]: what about `this.isCrossDomain= arguments[1].indexOf('//') > -1 && arguments[1].indexOf(window.location.origin) < 0`

Comment (dg) [inline src/Tracy/assets/Bar/bar.js]: And what about to send `setRequestHeader('X-Tracy-Ajax', layer.dataset.id)` directly in open() and leave send() unchanged?

**Comment by dg** [comment]
Thanks! Did you test it, does it work in your use case?

**Comment by tito10047** [comment]
yes. its work :)

**Comment by dg** [comment]
Great! Merged 1d8781dbcf5f210a315d83daf3a5c66db79de203

**Pull request closed without merging**